src/main/kotlin/frc/robot/subsystems/PickAndPlaceSubsystem.kt

```diff
@@ -0,0 +1,170 @@
+        elbowEncoder.velocityConversionFactor = 2.0 * Math.PI / 60.0
+
+        elevatorEncoder.positionConversionFactor = ArmConstants.elevatorEncoderPositionConversionFactor
+        elevatorEncoder.velocityConversionFactor = ArmConstants.elevatorEncoderVelocityConversionFactor
+
+        wristEncoder.positionConversionFactor = 2.0 *Math.PI
+        wristEncoder.velocityConversionFactor = 2.0 *Math.PI / 60.0
+        // TODO: wristEncoder position and velocity conversion factor
+    }
+
+    val elbowPositionRadians get() = elbowEncoder.position + ArmConstants.elbowEncoderPosOffset
+    val elevatorPositionMeters get() = elevatorEncoder.position
+
+    //intake
+    val relativeWristPosition get() = 2.0
+
+    /** This is the pitch with taking consideration to the position of the elbow.
+     * 0 degrees means the intake is parallel to the ground.
+     */
+    val absoluteWristPosition get() = 2.0
+
+    /** This value sets the voltage for the intake motors. Positive value will spin the wheels inward
+     * and pick objects up.
+     */
+    var intakesVoltage = 0.0
+        set(x: Double) {
+            field = x
+            intakeOneMotor.setVoltage(x)
+            intakeTwoMotor.setVoltage(x)
+        }
+
+    /** This value sets the voltage for the pitch motors. Positive values will tilt the intake up,
+     * negative values will tilt the intake down.
+     */
+    var wristVoltage = 0.0
+        set(x: Double) {
+            field = x
+            wristMotor.setVoltage(x)
+        }
+
+    var elbowVoltage = 0.0
+        set(x: Double) {
+            field = x
+            elbowMotor.setVoltage(x)
+        }
+
+    var elevatorVoltage = 0.0
+        set(x: Double) {
+            field = x
+            if (bottomHit && x < 0.0) {
+                elevatorMotor.setVoltage(0.0)
+            } else if (topHit && x > 0.0) {
+                elevatorMotor.setVoltage(0.0)
+            } else {
+                elevatorMotor.setVoltage(x)
+            }
+        }
+
+    override fun periodic() {
+        super.periodic()
+        // These are the voltages we set and will be sent to the elevator and elbow.
+        if(bottomHit)
+        {
+            elevatorEncoder.position = ArmConstants.elevatorMinHeight
+        }
+        else if(topHit)
+        {
+            elevatorEncoder.position = ArmConstants.elevatorMaxHeight
+        }
+
+        // Telemetry setting.
+        //Telemetry.elevatorVoltage.set(elevatorVoltage)
+        Telemetry.elbowPosition.set(elbowEncoder.position)
+        Telemetry.elevatorPosition.set(elevatorEncoder.position)
+        Telemetry.wristPosition.set(wristEncoder.position)
+        Telemetry.elbowVelocity.set(elbowEncoder.velocity)
+        Telemetry.topHit.set(topHit)
+        Telemetry.bottomHit.set(bottomHit)
+        Telemetry.wristVoltageNT.set(wristVoltage)
+        Telemetry.elbowVoltageNT.set(elbowVoltage)
+        Telemetry.elevatorVoltageNT.set(elevatorVoltage)
```

Comment:
get rid of the NT at the end of the names of these variables.

Why are some of the telemetry values commented out?